StRoot/StFttClusterMaker/StFttClusterMaker.h

```diff
@@ -0,0 +1,65 @@
+/***************************************************************************
+ *
+ * $Id: StFttClusterMaker.h,v 1.3 2019/02/19 20:32:05 fseck Exp $
+ *
+ * Author: Florian Seck, April 2018
+ ***************************************************************************
+ *
+ * Description: StFttClusterMaker - class to fill the StEvent from DAQ reader:
+ * unpack raw data & save StETofHeader & StETofDigis in StETofCollection 
+ *
+ ***************************************************************************/
+#ifndef STFTTCLUSTERMAKER_H
+#define STFTTCLUSTERMAKER_H
+#include "StMaker.h"
+#include <vector>
+#include <map>
+
+
+
+class StFttDb;
+class StEvent;
+class StFttCollection;
+class StFttRawHit;
+class StFttCluster;
+
+class StFttClusterMaker: public StMaker {
+
+public:
+    StFttClusterMaker( const char* name = "stgcCluster" );
+
+    ~StFttClusterMaker();
+
+
+    Int_t  Init();
```

Comment:
Coding standards prefer basic C++ types (int, double, char, etc...) over ROOT types (Int_t, Double_t, Char_t, etc...) for non-persistent classes.

Reply:
Addressed: 3686a6eea3b300c95180bdcee0ad698b5763a88c